OloEngine/src/OloEngine/Audio/SoundGraph/SoundGraphSource.cpp

```
@@ -328,6 +340,57 @@ namespace OloEngine::Audio::SoundGraph
+    void SoundGraphSource::UnregisterSpatializer()
+    {
+        if (!m_SpatializerRegistered || !m_Spatializer)
+            return;
+
+        m_Spatializer->ReleaseSource(m_SpatializerSourceID);
+        m_Spatializer = nullptr;
+        m_SpatializerSourceID = 0;
+        m_SpatializerRegistered = false;
+    }
+
+    void SoundGraphSource::UpdateSpatialPosition(const Audio::Transform& transform, glm::vec3 velocity)
```

Cppcheck (2.21.0)

[style] The function 'UpdateSpatialPosition' is never used.

(unusedFunction)